### DISCUSSION TITLE
Zerotier Networks don't get propagated

### DISCUSSION THREAD

**Discussion by cschmidtoptima** [sinamics/ztnet#559] [Q&A]
Hello,

I had to change the server where the zerotier container is running. I made a full server backup and deployed it to a new server. The controller is running and the api is available. However when I create a new network with e.g ztnet and try to join it, the client log says "not found"

When I deploy a fresh instance of ztnet on the same server everything works as expected, but  then I loose all my networks.

**Comment by PizzaProgram** [comment]
- Was is a self-hosted root planet ?
- Or the default official moon?

- If the first, can you move back temporary to the original server and add the new as secondary?

Anyway:
- how did you do a "full backup" ? (Did not find a button for it.)
